src/utils/helpers/formatDate.ts

```diff
@@ -0,0 +1,34 @@
+const MONTHS: Indexed = {
+  0: 'Jan',
+  1: 'Feb',
+  2: 'Mar',
+  3: 'Apr',
+  4: 'May',
+  5: 'Jun',
+  6: 'Jul',
+  7: 'Aug',
+  8: 'Sep',
+  9: 'Oct',
+  10: 'Nov',
+  11: 'Dec',
+};
+
+const formatDate = (date: string | undefined): string => {
```

Comment:
Тут можно написать так: 
```suggestion
const formatDate = (date: string = ''): string => {
```

И тогда ты избавишься от if'a внутри функции